tests/forte/data/audio_annotation_test.py

```diff
@@ -27,18 +28,30 @@
 from forte.data.data_pack import DataPack
 from forte.data.readers import AudioReader
 from forte.data.ontology.top import (
-    Annotation, AudioAnnotation, Generics, Group, Link
+    Annotation,
+    AudioAnnotation,
+    Generics,
+    Group,
+    Link,
+)
+from ft.onto.base_ontology import (
+    Recording,
+    AudioUtterance,
+    TextPayload,
+    Utterance,
 )
-from ft.onto.base_ontology import Recording, AudioUtterance, Utterance
 
 
 class RecordingProcessor(PackProcessor):
     """
     A processor to add a Recording ontology to the whole audio data.
     """
+
     def _process(self, input_pack: DataPack):
         Recording(
-            pack=input_pack, begin=0, end=len(input_pack.audio)
+            pack=input_pack,
+            begin=0,
+            end=len(input_pack.get_payload_data_at(Modality.Audio, 0)),
```

Comment:
This change might also be unnecessary since `input_pack.audio` is doing the same thing here.

Reply:
I think Hector mentioned that we want to remove `audio` and `text` property in datapack. In this PR I have already removed `audio` property. Since there are too many code dependent on `text` property, we want to remove it in a separate PR.